src/widgets/query-rule-custom-data/__tests__/query-rule-custom-data-test.ts

```diff
@@ -54,6 +54,17 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/query-rule-
 See documentation: https://www.algolia.com/doc/api-reference/widgets/query-rule-custom-data/js/"
 `);
     });
+
+    it('is a widget', () => {
+      const container = document.createElement('div');
+      const widget = queryRuleCustomData({ container });
+
+      expect(widget).toEqual(
+        expect.objectContaining({
+          $$type: 'ais.queryRuleCustomData',
```

Comment:
What about `ais.queryRule.customData`? Kinda like the connect pattern on display name in react

Reply:
I think I'd rather see another `$$connectorType` attribute when we need it in the future.